client/blocks/reader-full-post/_content.scss

```
@@ -27,13 +38,16 @@
 	}
 
 	h4 {
-		font-size: 18px;
-		font-weight: 700;
+		font-size: 32px;
```

With the introduction of these: 

https://github.com/Automattic/wp-calypso/blob/a83f9acf2b30fc352da61669616f280df655134b/packages/typography/styles/variables.scss#L39

I thought I should offer this suggestion since there is a variable that matches the pixel value in the PR.

```suggestion
		font-size: $font-title-large;
```